library/api_legacy_filter.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"C"
+
+	mobile "github.com/waku-org/go-waku/mobile"
+)
+
+// Creates a subscription to a light node matching a content filter and, optionally, a pubSub topic.
```

Comment:
Better to start marking legacyFilter API's as deprecated indicating that they would be removed in upcoming releases.